calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -131,26 +147,41 @@ def used_cucumber?
       #
       # Collect a hash of usage info.
       def info
-        {
+
+        allowed = allowed_to_track
+
+        return {} if allowed == "none"
```

Comment:
Shouldn't this maybe return `nil` instead of `{}`? We don't want to send empty hashes up to node, because we'll end up with garbage data in MixPanel. 

Unless, of course, we're covertly interested in tracking how many people do not want to be tracked :)